core/src/term/mod.rs

```diff
@@ -926,18 +955,20 @@ pub enum UnaryOp {
 
     /// Map a function on each element of an array.
     ArrayMap(),
+
     /// Map a function on a record.
     ///
     /// The mapped function must take two arguments, the name of the field as a string, and the
-    /// content of the field. `RecordMap` then replaces the content of each field by the result of the
-    /// function: i.e., `recordMap f {a=2;}` evaluates to `{a=(f "a" 2);}`.
+    /// content of the field. `RecordMap` then replaces the content of each field by the result of
+    /// the function: i.e., `recordMap f {a=2;}` evaluates to `{a=(f "a" 2);}`.
     RecordMap(),
 
     /// Inverse the polarity of a label.
     ChangePolarity(),
 
     /// Get the polarity of a label.
     Pol(),
+
```

Comment:
It's not tremendously important, but I wonder if adding new lines everywhere makes it more readable: op used to be grouped somehow by theme (e.g `StrXXX`), but now there is no structure anymore. That being said the grouping was somehow historical.

Interestingly I wondered if inserting new lines between variants or fields was a common practice, thus I went to check in the source code of `rustc`. It seems that there is no logic there - sometimes enums and structs don't have a newline between fields and variants, sometimes they do. There seems to be, at least empirically, a notion of size - large structs/enums tend to be split.

Reply:
> I wonder if adding new lines everywhere makes it more readable

Definitely does for me, at least when each "item" is more than one line.

> Interestingly I wondered if inserting new lines between variants or fields was a common practice

You're not likely to find guidance on this, since rust APIs don't tend to expose the internals of their types. Grouping is also mostly fine, but in my experience it stops making sense rather quickly. If you wanted to variants to be that similar, you'd have merged them into one anyway.

Reply:
> You're not likely to find guidance on this, since rust APIs don't tend to expose the internals of their types. 

I was curious about any kind of unspoken standard, even if used only internally inside rustc. As for the `use` comment it's a detail anyway. I'm fine with the current way